.taskcat.yml

```diff
@@ -14,7 +14,6 @@ project:
     NumberOfAZs: 3
     KeyPairName: --override--
     RemoteAccessCidr: --override--
-    S3TAPConfigBucketName: $[taskcat_random-string]-$[taskcat_current_region]
```

Comment:
I didn't find any evidence we this parameter is used anywhere, so I took the liberty to remove it.

Reply:
it is not used now.